lib/utilities/relative-module-paths.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const path = require('path');
+const ensurePosix = require('ensure-posix-path');
+const { moduleResolve } = require('amd-name-resolver');
+
+const BASE_DIR = path.resolve(`${__dirname}/..`);
+
+function getRelativeModulePath(modulePath) {
+  return ensurePosix(path.relative(process.cwd(), modulePath));
+}
+
+function resolveRelativeModulePath(name, child) {
+  return moduleResolve(name, getRelativeModulePath(child));
+}
+
+Object.keys(module.exports).forEach(key => {
+  module.exports[key].baseDir = () => BASE_DIR;
+
+  module.exports[key]._parallelBabel = {
+    requireFile: __filename,
+    useMethod: key,
+  };
+});
```

Comment:
I'm wondering if we should move this into `amd-name-resolver` to not copy it around everywhere 🤔

Reply:
Yeah, seems reasonable to me

Reply:
Should we land this and then extract, or extract then land?

Reply:
I'm totally fine with landing and extracting later